stats/misc_stats1.php

```diff
@@ -84,10 +84,10 @@ function show_top_days( $n, $when )
         select_from_site_past_tallies_and_goals(
             $tally_name,
             "SELECT
-                {date} as '" . mysqli_real_escape_string(DPDatabase::get_connection(), _("Date")) . "',
-                tally_delta as '" . mysqli_real_escape_string(DPDatabase::get_connection(), _("Pages Proofread")) . "',
+                {date} as '" . DPDatabase::escape(_("Date")) . "',
```

Comment:
We localize the column aliases? This does seem odd. Do we raw dump these results somewhere to be shown in the UI?

Reply:
Yeah, these dump out to a table with the fields as column names. It's a long-standing and horrible design I don't want to solve here.